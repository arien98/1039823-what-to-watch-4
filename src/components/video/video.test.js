import React from "react";
import renderer from "react-test-renderer";
import Video from "./video.jsx";

describe(`main renders correctly`, () => {
  test(`it renders correctly`, () => {
    const props = {
      poster: `../img/aviator.jpg`,
      preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    };

    const tree = renderer
      .create(<Video {...props}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
