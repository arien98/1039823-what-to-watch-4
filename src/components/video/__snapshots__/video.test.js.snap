// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main renders correctly it renders correctly 1`] = `
<video
  autoPlay={true}
  height="175"
  muted={true}
  poster="../img/aviator.jpg"
  width="280"
>
  <source
    src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  />
</video>
`;
